packages/edit-post/package.json

```diff
@@ -22,10 +22,6 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
-	"sideEffects": [
```

Comment:
This package has a whole bunch of components, and this way none of them are modularised 😞 I *think* that it loads all of them anyway, so it should be fine, but it's hard to tell. And since there's an `initialize` method, I believe this could be done without side effects here too.

Reply:
Do you mean components like `PluginBlockSettingsMenuItem` that are reexported by `edit-post/src/index.js`? These are not really components. It's a plugin API that exposes some areas in UI where you can insert your own UI. More precisely, it's a `Fill` component that inserts something into a sibling `Slot`, where the pair is created by a `createSlotFill` call.

If you don't use these pseudo-components, it doesn't mean they can be removed from the bundle. The corresponding `Slot`s are still there.

Reply:
Oh, you're right, I missed the re-exports at the end of the file, my bad 👍